site2/docs/schema-evolution-compatibility.md

```diff
@@ -30,19 +30,34 @@ For more information, see [Schema compatibility check strategy](#schema-compatib
 
 ### How does Pulsar support schema evolution?
 
-1. When a producer/consumer/reader connects to a broker, the broker deploys the schema compatibility checker configured by `schemaRegistryCompatibilityCheckers` to enforce schema compatibility check. 
+The process of how Pulsar supports schema evolution is described as follows.
 
-   The schema compatibility checker is one instance per schema type. 
+1. The producer/consumer/reader sends its client `SchemaInfo` to brokers. 
    
-   Currently, Avro and JSON have their own compatibility checkers, while all the other schema types share the default compatibility checker which disables schema evolution.
+2. Brokers recognize the schema type and deploy the schema compatibility checker `schemaRegistryCompatibilityCheckers` for that schema type to enforce schema compatibility check. 
```

Comment:
I suggest adding `schemaRegistryCompatibilityCheckers` config here, so like:
```
schemaRegistryCompatibilityCheckers=org.apache.pulsar.broker.service.schema.JsonSchemaCompatibilityCheck,org.apache.pulsar.broker.service.schema.AvroSchemaCompatibilityCheck,org.apache.pulsar.broker.service.schema.ProtobufNativeSchemaCompatibilityCheck
```

Reply:
You also need to add `schemaRegistryCompatibilityCheckers` to https://pulsar.apache.org/docs/next/reference-configuration.

Reply:
Need to double-check the default compatibility checker for other schema types. Seems it's not shown in the default value.